implementation/booting-rust.tex

```diff
@@ -17,14 +17,14 @@ \section{Booting Rust on the Gecko}
 
 In \autoref{lst:impl:c++:static-constructor} the Foo constructor call must be issued before the main function is executed.
 The runtime must therefore be augmented to issue calls to these constructors and handle errors that can occure.
-These types of constructors are not allowed in Rust by a compiler rule which allows only constant-expression as initializers.
+These types of constructors are not allowed in {\rust} by a compiler rule which allows only constant-expression as initializers.
 (An interesting side note here is given in \autoref{sec:impl:lazy-statics}.)
 
 As this process is identical to C we are able to reuse standard runtime components for embedded toolchains for C.
-SiliconLabs software suite provides the \textbf{ResetHandler} and linker script for the efm32 microcontrollers.
-The \textbf{\_mainCRTStartup} function defined in the C runtime provided by the \textbf{arm-none-eabi} gcc toolchain handles the setup in \textbf{\_start}:
+SiliconLabs software suite provides the \func{ResetHandler} and linker script for the efm32 microcontrollers.
+The \func{\_mainCRTStartup} function defined in the C runtime provided by the \textbf{arm-none-eabi} gcc toolchain handles the setup in \func{\_start}:
 
-\subsection{Minimal Rust program to boot}
+\subsection{Minimal {\rust} program to boot}
 
 There are a few modification which have to be applied to the default 'Hello World' program in order to get it to boot in an embedded environment.
```

Comment:
Quotes er annerledes i latex,  bruk 
*  `Hello world', eller
* ``Hello World''

Altså forskjellige fnutter for start og slutt